scripts/src/main/resources/scripts/command/az

```diff
@@ -1,106 +1,77 @@
 #!/usr/bin/env bash
-
-# autocompletion list
-if [ "${1}" = "shortlist" ]
-then
-  if [ -z "${2}" ]
-  then
-    echo "setup help"
-  fi
-  exit
-fi
-
-if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
 
 function doSetup() {
-  if command -v az &> /dev/null
+  setLocation
+  if [ "${1}" != "silent" ] || [ ! -f "${AZ_FILE}" ] 
   then
-    if [ "${1}" != "silent" ] && ! doIsQuiet
+    if [ -d "${PYTHON_HOME}" ] 
     then
-      doEcho "Azure CLI is already installed at $(command -v az)"
+      doDevonCommand python pip install azure-cli
+      setConfig
+    else
+      doDevonCommand python setup silent
+      doDevonCommand python pip install azure-cli
+      setConfig
     fi
-  else
-    doRequireNotBatch
-    if doIsWindows
-    then
-      # Get leatest release
-      if [ -z "${AZ_VERSION}" ]
-      then
-        doEcho "Getting latest release..."
-        AZ_VERSION=$(curl -s https://api.github.com/repos/Azure/azure-cli/releases/latest | awk -F ":" '/tag_name/ {print $2}'| awk -F "\"" '{print $2}' | awk -F "-" '{print $3}')
-      fi
+  fi
+  if [ "${1}" != "silent" ] && ! doIsQuiet
+  then
+    doRunCommand "az -v" "verify installation of azure cli"
+  fi
+}
 
-      doInstall "-" "${AZ_HOME}" "az" "${AZ_VERSION}"
-      if [ "${?}" = 0 ]
-      then
-        doPackageInstall "${AZ_HOME}"
-      fi
-    fi
+function setConfig() {
+  az_config_dir="${DEVON_IDE_HOME}/conf/.azure"
+  az_config_export="export AZURE_CONFIG_DIR=${az_config_dir}"
+  if ! grep -q "${az_config_export}" "${DEVON_IDE_HOME}/conf/devon.properties"
+    then
+      doRunCommand "${az_config_export}"
+      echo -e "\n${az_config_export}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
+      doEcho "Location of Azure's configuration file is set to ${az_config_dir}"
   fi
 }
 
-function doPackageInstall() {
-  local path_to_package="${1}"
+function setLocation() {
   if doIsWindows
   then 
-    local winPath
-    winPath=$(cygpath -w "${path_to_package}")
-    powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${winPath}\az-${AZ_VERSION}-windows.msi /QB-!'" &> /dev/null
-    if [ "${?}" = 0 ]
-    then
-      setConfigDir
-      doSuccess "Azure CLI ist installed now. Great! \nTo get started you may need to restart your current shell."
-    else
-      doError "The setup was canceled or failed. Rerun the setup to install Azure CLI."
-    fi
+    AZ_FILE="${DEVON_IDE_HOME}/software/python/Scripts/az"
```

Comment:
Actually it is not so good to have internal dependencies and knowledge of python/pip inside this azure commandlet.
As @BizarreBits changed #795 and there is no `Scripts` folder anymore this will not work anymore.
Actually we should assume that everything will automatically be added to the `PATH` so you could directly call it.

Tricky would be to distinguish if the `az` installation comes from a global installation or from the local IDE.
I would suggest to use `command -v az` and then check that the result of this starts with `${DEVON_IDE_HOME}/software/`.

Reply:
I use the `command -v az` option, but the problem is still that the local az script is not automatically added in the path.

Reply:
The az script is still located at `software/python/Scripts`. Pip create this folder automatically during package installations. Either we need to add the script folder to the path or we try to use a target switch as described here? https://stackoverflow.com/a/19404371

Reply:
> I use the `command -v az` option, but the problem is still that the local az script is not automatically added in the path. The behavior I assume is that after I run `devon`, `command -v az` shows me the local path. Currently the global path is output like `/c/Program Files (x86)/Microsoft SDKs/Azure/CLI2/wbin/az.`
> 
> Furthermore, it would be important that when Python is also installed with `doDevonCommand python pip install azure-cli,` the environment variables are updated before`pip install azure cli` is executed, otherwise the setup will abort.

Could be fixed with by adding `PATH=/c/projects/devonfw-ide-scripts-2022.04.003/software/python/Scripts:$PATH` to devon.properties

Reply:
Same problem as discussed in the thread below.
We will fix this properly and can stop hardcoding paths and knowledge about tools that belong to other commandlets.
The right way to go is always to ensure that all binaries are either in the `bin` subfolder or if not present in the tool folder itself. If that is not met, we actually have a bug and tweaking the PATH with manual hacks will not solve it properly but only cause later problems somewhere else.

Reply:
already obsolete and resolved.